gradle.properties

```diff
@@ -1,4 +1,4 @@
-VERSION_NAME=2.0.11-SNAPSHOT
+VERSION_NAME=2.1.1-SNAPSHOT
```

Comment:
Why are we jumping to 2.1.1 instead of 2.1.0?

Reply:
Reverted to v2.1.0